src/components/Header.jsx

```diff
@@ -9,7 +9,6 @@ import { selectUser } from "selectors/authenticate"
 import { signOut } from "actionCreators/authenticate"
 import { bindActionCreators } from "redux"
 import { selectCurrentResourceKey } from "selectors/resources"
-import { selectAppVersion } from "selectors/index"
 import usePermissions from "hooks/usePermissions"
```

Comment:
these statements in Header should have been removed in PR #3107, as they stopped being used because of that PR.